ihatemoney/migrations/versions/2dcb0c0048dc_autologger.py

```diff
@@ -28,6 +28,7 @@ def upgrade():
             "external_link", sa.UnicodeText(), autoincrement=False, nullable=True
         ),
         sa.Column("archive", sa.Integer(), autoincrement=False, nullable=True),
+        sa.Column("is_reimbursement", sa.Boolean(), nullable=True),
```

Comment:
This file is auto-generated, please read https://ihatemoney.readthedocs.io/en/latest/contributing.html#creating-database-migrations